using-valtimo/plugin/zaken-api/configure-zaken-api-plugin.md

```diff
@@ -83,3 +83,17 @@ When creating a process link the following properties have to be entered:
 - **Other natural person identification.** (Optional) The unique number issued by the municipality for another natural
   person.
 - **Administration number person.** (Optional) The administration number of the person, as referred to in the BRP Act.
+
+### Start recovery period
+
+The **Start recovery period** will start a recovery period for the linked zaak in the zaken API. If a zaak has a due
+date (Uiterlijke einddatum afdoening) set, the due date will be extended with the configured maximum number of days that
+the recovery period will last. The zaak will also be suspended.
+
+- **Maximum duration in days.** The maximum duration of the recovery period in days.
+
+### End recovery period
+
+The **End recovery period** will end a recovery period for the linked zaak in the zaken API. The due
+date (Uiterlijke einddatum afdoening) of the zaak will be extended by the actual duration of the recovery period. The
```

Comment:
Maybe we want to explain that this results in a reduction?

Reply:
I've added an explaination